src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java

```diff
@@ -959,4 +965,14 @@ private void applyUplinkRecirculation(Ip4Prefix subnet, boolean remove) {
             flowRuleService.removeFlowRules(denyRule, allowRule);
         }
     }
+
+    private boolean gettableFlowEntry(FlowEntry flowEntry) {
```

Comment:
Few alternatives that would improve readability, I think:
```suggestion
    private boolean isHereToStay(FlowEntry flowEntry) {
```
```suggestion
    private boolean inserted(FlowEntry flowEntry) {
```